lib/maril/generator.rb

```diff
@@ -19,7 +19,7 @@ def generate
       end
       cmd << app['container']['docker']['image']
       cmd << app['cmd'] if app['cmd']
-      cmd.join(" \\\n")
+      cmd.join(" ")
```

Comment:
I agree with the readability, I do think it's better when separated by lines. 

But this is what happens when I paste a really long command (e.g. for `/development/api`) into my terminal:
<img width="1278" alt="screen shot 2017-08-03 at 5 13 40 pm" src="https://user-images.githubusercontent.com/2593686/28949098-4bc3b51a-786f-11e7-9a2b-e4f5cdba9ee4.png">

The arguments get all mashed together, and at some point it thinks I've hit enter, and stuff fails.